eventmesh-registry-plugin/eventmesh-registry-zookeeper/src/main/java/org/apache/eventmesh/registry/zookeeper/service/ZookeeperRegistryService.java

```diff
@@ -0,0 +1,306 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Same as above!

Reply:
@VNBear Some licenses are not in the right format, please check!

Reply:
done, please approval again, thk !